help/submit.md

```diff
@@ -1,17 +1,18 @@
 # Submission Guidelines
 
 
-<span id="guidelines"></span> Submissions to arXiv should be topical and refereeable scientific contributions that follow accepted standards of scholarly communication.
+<span id="guidelines"></span> While submission to arXiv is free for authors, we do ask authors to carefully prepare their work according to these guidelines. This will improve discoverability of the work and reduce the likelihood of delays before announcement.

```

Comment:
```suggestion
<span id="guidelines"></span> 

While submission to arXiv is free for authors, we do ask authors to carefully prepare their work according to these guidelines. This will improve discoverability of the work and reduce the likelihood of delays before announcement.

```

Reply:
some browsers display what is below an anchor - if the anchor is inline with text that text may be above the page display